src/app/app.component.ts

```diff
@@ -7,7 +7,11 @@ import { Component } from '@angular/core';
   selector: 'aou-app',
   styleUrls: [ './app.component.css' ],
   template: `
-    <h1>{{title}}</h1>
+    <div style="display: flex; align-items: flex-end;">
+      <img src="images/all-of-us-logo.svg" alt="All of Us Logo" style="width: 18rem;">
```

Comment:
How about {{title}} as the alt?

My approach is to use the alt text not necessarily as a description of the image, but as a replacement for it -- so in this case, I assume the SVG has text similar to {{title}}, and using {{title}} in its place for screen readers would be a reasonable substitution.